cmd/root.go

```diff
@@ -50,8 +50,20 @@ const (
 	msgSize = 512 * 1024 * 1024
 )
 
-var encodings = []string{"json", "bytes", "proto", "ascii", "json_ietf"}
-var formats = []string{"json", "protojson", "prototext", "event", "proto"}
+var encodings = [][2]string{
+	{"json", "indicates that the value is encoded as a JSON string,as specified in RFC7159"},
+	{"bytes", "contains a byte sequence whose semantics is opaque to the protocol"},
+	{"proto", "contains a serialised protobuf message using protobuf.Any"},
+	{"ascii", "an ASCII encoded string representing text formatted according to a target-defined convention"},
+	{"json_ietf", "similar to `json` with aditional types defined"},
+}
+var formats = [][2]string{
+	{"json", "similar to protojson but with xpath style paths and decoded timestamps"},
+	{"protojson", "protocol buffer messages as JSON format"},
+	{"prototext", "protocol buffer messages as the textproto format"},
+	{"event", "protocol buffer messages as a timestamped list of tags and values"},
+	{"proto", "protocol buffer messages as the binary wire format"},
```

Comment:
`as -> in`?

protocol buffer messages as JSON format -> protocol buffer messages in JSON format
reads more naturally to me

Reply:
copied the package description from here: https://godoc.org/google.golang.org/protobuf/encoding/protojson